### DISCUSSION TITLE
How to open vim from fxtui?

### DISCUSSION THREAD

**Discussion by dangooddd** [ArthurSonzogni/FTXUI#954] [Q&A]
That's it, how can I do that? If i ran boost::process or std::system, nvim blocks, and after i exit nvim program crushes. Please help!

**Comment by ArthurSonzogni** [comment]
Hello, you might want to wrap your function executing (and waiting) vim to execute inside:
https://github.com/ArthurSonzogni/FTXUI/blob/70bc44d28b36bc7b7332a88e19e9ad99c270074b/include/ftxui/component/screen_interactive.hpp#L60

Example:
https://github.com/ArthurSonzogni/FTXUI/blob/70bc44d28b36bc7b7332a88e19e9ad99c270074b/examples/component/with_restored_io.cpp#L23

**Reply by dangooddd** [reply]
Thanks!
